pkg/asset/manifests/gcp/cluster.go

```diff
@@ -33,26 +38,44 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 		networkName = installConfig.Config.GCP.Network
 	}
 
-	masterSubnet := gcp.DefaultSubnetName(clusterID.InfraID, "master")
+	masterSubnetName := gcp.DefaultSubnetName(clusterID.InfraID, "master")
+	masterSubnetCidr := ""
 	if installConfig.Config.GCP.ControlPlaneSubnet != "" {
-		masterSubnet = installConfig.Config.GCP.ControlPlaneSubnet
+		masterSubnetName = installConfig.Config.GCP.ControlPlaneSubnet
+
+		masterSubnet, err := getSubnet(installConfig.Config.GCP.NetworkProjectID, installConfig.Config.GCP.Region, masterSubnetName)
+		if err != nil || masterSubnet == nil {
+			return nil, fmt.Errorf("failed to get master subnet: %w", err)
+		}
+		// IpCidr is the IPv4 version, the IPv6 version can be accessed as well
+		masterSubnetCidr = masterSubnet.IpCidrRange
 	}
+	logrus.Warnf("MASTER SUBNET NAME: %s, CIDR: %s", masterSubnetName, masterSubnetCidr)
```

Comment:
This looks like a dev debug line. I suggest either wording it better or removing for the final version of the PR.